beacon_node/eth2-libp2p/src/rpc/handler.rs

```diff
@@ -600,6 +584,7 @@ where
                                         }
                                         entry.remove_entry();
                                         return Ok(Async::Ready(ProtocolsHandlerEvent::Custom(
+                                            // TODO: remove hardcoded RequestId
```

Comment:
This is to do with the first dot point of this issue: https://github.com/sigp/lighthouse/issues/987